crates/ln-dlc-node-tests/src/tests/dlc/collaborative_settlement.rs

```diff
@@ -1,9 +1,8 @@
-use crate::node::InMemoryStore;
-use crate::node::Node;
 use crate::tests::dlc::create::create_dlc_channel;
 use crate::tests::init_tracing;
 use crate::tests::wait_until_dlc_channel_state;
 use crate::tests::SubChannelStateName;
+use crate::tests::TestNode;
```

Comment:
:question: Can we just call it `Node`?

Reply:
we could, but because we have many `Node`s , I would need to start qualifying it in many places; it was easier to use a different name.